proms-resources/frontend/src/main/frontend/src/TOU.json

```
@@ -0,0 +1,80 @@
+{
+  "title" : "EPRO Terms of Use",
+  "text" : [
+    "These terms and conditions (the \"Agreement\") for a symptoms survey located at ePRO.uhn.ca and any associated website, portal, or similar service (collectively, the \"Application\"), constitute a legal agreement and is entered into by and between you (\"you\", \"your\") and the University Health Network (\"UHN\", \"we\", \"us\", \"our\").\\\n",
+    "**IMPORTANT! BY USING OR ACCESSING THE APPLICATION, YOU AGREE TO BECOME BOUND BY THE AGREEMENT. IF YOU DO NOT AGREE WITH THE AGREEMENT, YOU MUST NOT USE OR ACCESS THE APPLICATION. NOTE THAT THERE ARE ALTERNATIVE METHODS TO COMPLETE THE SURVEY IF YOU WOULD NOT LIKE TO PROCEED WITH THIS APPLICATION.**\\\n",
+    "This Agreement governs your access to the Application and use, including any content, functionality, and services offered on or through the Application.\\\n",
+    "The Application is intended to act as an information collection tool for physicians to better prepare for appointments with patients and to improve record keeping. The Application is intended to facilitate symptom reporting to healthcare providers, for quality monitoring and improvement, and general patient education relative to cancer symptom status. It is not a medical device. Nothing contained in the Application should be construed, directly or indirectly, as the practice of medicine or providing medical advice or medical services by UHN. The Application is not intended to diagnose, cure, mitigate, treat, or prevent disease or other conditions.\\\n",
+    "####Not Medical Advice.\\\n",
+    "The App is not a substitute for professional medical advice, diagnosis or treatment. Never make changes to your medication, nor adjust your dose, without first consulting your health care provider. Always seek the advice of a physician or other qualified healthcare provider concerning questions you have regarding a medical condition, and before starting, stopping or modifying any treatment or medication. Never delay obtaining medical advice or disregard medical advice because of something you have or have not read on the App. If you have, or suspect you have, a health problem, or if you experience an adverse side effect, please consult your doctor. If you have, or suspect you are experiencing a health emergency, please call 911 and/or promptly visit a Hospital Emergency Department in your area.\\\n",
+    "####No Regulatory Approval.\\\n",
+    "The App has not been evaluated by Health Canada, the United States Food and Drug Administration (**USFDA**) or any other domestic or foreign regulatory agency and is not approved to diagnose, treat, cure or prevent any disease or for the prescription of any medication or treatment. While the information contained may be of use in other jurisdictions, these materials were provided and designed for use in Canada and you are solely responsible for ensuring suitability and permitted use in other jurisdictions.\\\n",
```

According to the original text, these look like they're supposed to be paragraph titles, on the same line.
```suggestion
    "**No Regulatory Approval.** The App has not been evaluated by Health Canada, the United States Food and Drug Administration (**USFDA**) or any other domestic or foreign regulatory agency and is not approved to diagnose, treat, cure or prevent any disease or for the prescription of any medication or treatment. While the information contained may be of use in other jurisdictions, these materials were provided and designed for use in Canada and you are solely responsible for ensuring suitability and permitted use in other jurisdictions.\\\n",
```

---

Done in 8c43d6e